src/Sulu/Component/Content/Metadata/Loader/XmlLoader.php

```diff
@@ -34,7 +35,10 @@ public function load($resource, $type = 'page')
 
         $structure = new StructureMetadata();
         $structure->name = $data['key'];
-        $structure->cacheLifetime = $data['cacheLifetime'];
+        $structure->cacheLifetime = new CacheLifetimeMetadata(
```

Comment:
i would love to see a validation here you could introduce  a `validation` function in the `CacheLifetimeResolver` and call it here